src/pages/pricing/index.tsx

```diff
@@ -56,10 +57,8 @@ export default function Pricing() {
       <Head>
         <title>Pricing | PrairieLearn</title>
       </Head>
+      <PageBanner title="Pricing" text="" />;
```

Comment:
Once you make the secondary prop optional, you can remove it entirely here.
```suggestion
      <PageBanner title="Pricing" />;
```